pkg/services/forwarder/ports.go

```diff
@@ -187,3 +187,11 @@ func (f *PortsForwarder) Mux() http.Handler {
 	})
 	return mux
 }
+
+func eventuallyAddMissingIP(req types.ExposeRequest, ip string) string {
+	if strings.HasPrefix(req.Remote, ":") {
+		split := strings.Split(ip, ":")
```

Comment:
This should use `net.SplitHostPort` I think

Reply:
done 👍